.nvmrc

```
@@ -1 +1 @@
-v16.13.0
\ No newline at end of file
+v18.19.1
```

Probably shouldn't be hardcoded to a specific patch and minor version in order to ensure security updates are applied?
```suggestion
18
```

---

Maybe I can pin 18.19? Pnpm in the latest version requires at least node 18.17.x :thi

---

Vercel will decide which minor anyway:
https://vercel.com/docs/functions/runtimes/node-js#default-and-available-versions

---

I would opt for major + minor. Node 18 is currently in maintenance mode till end of 2024 so they release mostly patches. Having both major & minor will help with local development - it will instruct fnm or nvm to fetch version larger than 18.17.